### PR TITLE
Add RabbitMQ support

### PR DESCRIPTION
**Pull request by ibrt**

### CODE REVIEW

File: rabbitmq.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		forward = "127.0.0.1:" + forward
+	}
+	c, ip, err = SetupContainer(RabbitMQImageName, port, 10*time.Second, func() (string, error) {
+		res, err := run("--name", GenerateContainerID(), "-d", "-P", "-p", forward, RabbitMQImageName)
+		return res, err
+	})
+	return
+}
+
+// Connect starts a RabbitMQ image and passes the amqp url to the connector callback.
```
Comment (arekkas) [inline rabbitmq.go]:
```
// Connect starts a RabbitMQ image and ...
```

should be

```
// ConnectToRabbitMQ starts a RabbitMQ image and ...
```

Comment (ibrt) [inline rabbitmq.go]: Fixed

**Comment by ibrt** [comment]
Fixed, thanks :)

**Comment by arekkas** [comment]
Perfect, thank you!

**Pull request merged**